clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -108,6 +108,10 @@ Changes in existing checks
   <clang-tidy/checks/readability/redundant-smartptr-get>` check to
   remove `->`, when reduntant `get()` is removed.
 
+- Added option `UseUpperCaseLiteralSuffix` to :doc:`readablility-implicit-bool-conversion
```

Comment:
Please follow above conversion, simply because its later easier to find check name.
Check release notes from previous release.